src/test/unit/rcdevice_unittest.cc

```diff
@@ -118,6 +115,16 @@ static void clearResponseBuff()
     }
 }
 
+static void resetRCDeviceStatus()
+{
+    isButtonPressed = false;
+    rcdeviceInMenu = false;
+    pgResetFn_rcdeviceConfig(rcdeviceConfigMutable());
```

Comment:
PG_RESET(rcdeviceConfig); should work and is cleaner

Reply:
ok, fixed